run.py

```diff
@@ -1,10 +1,10 @@
 from votey import create_app
-from votey import db
+from votey.exts import db
 
 app = create_app()
 
 with app.app_context():
     db.create_all()
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=5050, threaded=True)
+    app.run(host="127.0.0.1", port=5050, threaded=True)
```

Comment:
I don't recall why we do `0.0.0.0`, but this won't break heroku, right?

Reply:
this should come up in testing if it does. i changed it because bandit flagged it as a potential security risk, and i think even if it does break, i should be able to satisfy bandit by explicitly picking an interface to bind to in heroku and maybe making that a variable in the .env